implementors.md

```diff
@@ -0,0 +1,10 @@
+# OCI Artifacts Implementations
+
+Projects or companies currently supporting OCI Artifacts
```

Comment:
maybe add... "as previously specified under the OCI Image and Runtime specifications (links)?

Reply:
While I need to update docker/distribution to be distribution/distribution, are we ok to merge this in? This will be the base for others to add their support.

Reply:
sure let's just make it distribution/distribution then merge..